tests/client/amo/test_utils.js

```diff
@@ -0,0 +1,69 @@
+import createStore from 'amo/store';
+import * as landingActions from 'amo/actions/landing';
+import * as api from 'core/api';
+import { loadLandingAddons } from 'amo/utils';
+
+
+describe('AMO utils loadLandingAddons()', () => {
+  const addonType = 'theme';
+  let ownProps;
+
+  before(() => {
+    ownProps = {
+      params: {
+        addonType: 'themes',
+        application: 'android',
+      },
+    };
+  });
+
+  it('returns right away when loaded', () => {
+    const store = createStore({ application: 'android' });
+    store.dispatch(landingActions.getLanding({ addonType }));
+    const mockApi = sinon.mock(api);
+    const entities = sinon.stub();
+    const result = sinon.stub();
+
+    mockApi
+      .expects('featured')
+      .once()
+      .withArgs({ api: {}, filters: { addonType, page_size: 4 } })
+      .returns(Promise.resolve({ entities, result }));
+    mockApi
+      .expects('search')
+      .once()
+      .withArgs({
+        api: {},
+        filters: { addonType, sort: 'rating', page_size: 4 },
+        page: 1,
+      })
+      .returns(Promise.resolve({ entities, result }));
+    mockApi
+      .expects('search')
+      .once()
+      .withArgs({
+        api: {},
+        filters: { addonType, sort: 'hotness', page_size: 4 },
+        page: 1,
+      })
+      .returns(Promise.resolve({ entities, result }));
+
+    return loadLandingAddons({ store, params: ownProps.params })
+      .then(() => {
+        mockApi.verify();
+        assert.strictEqual(loadLandingAddons({
+          store: {
+            dispatch: sinon.stub(),
+            getState: () => ({
```

Comment:
this seems wrong. It should be returning `state.landing.addonType...` (I think?). Anyway, why do you need a stub store? Can't you just use the real one?